scripts/wp-plugin.js

```diff
@@ -7,32 +7,46 @@ const path   = require('path');
 const yaml   = require('js-yaml');
 const config = yaml.safeLoad(fs.readFileSync(path.resolve(__dirname, '..', 'config', '_config.yml'), 'utf8'));
 
+const OUT_DIR = path.resolve(__dirname, '..', 'data');
+
 const data = {
     timestamp: new Date(),
     bootstrap: {},
     fontawesome: {}
 };
 
-config.bootstrap.forEach((bootstrap) => {
+config.bootstrap4.forEach((bootstrap) => {
+    data.bootstrap[bootstrap.version] = {
+        css: bootstrap.stylesheet,
+        js: bootstrap.javascript,
+        jsBundle: bootstrap.javascriptBundle
```

Comment:
Only thing I don't know is if the WP plugin will have issues with this addition.

Reply:
@petarpetrovic can you confirm?